.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,7 +11,7 @@ Add resource(s) | Remove resource(s) | Add info | Improve repo
 ### For book lists, is it a book? For course lists, is it a course? etc.
 
 ## Checklist:
-- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
+- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/CONTRIBUTING.md)
```

Comment:
```suggestion
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
```

This file has been moved (#6614)

Reply:
Yes, you are right. I miss on https://github.com/EbookFoundation/free-programming-books/pull/6627/commits/4d3c9fe11c84604c09d4b3834a59cd5820bbd1f9